### PR TITLE
Remove PostgreSQL image workaround

### PR DESCRIPTION
**Pull request by jsmrcka**
### Summary

Removes workaround for a failure reported in
https://github.com/quarkus-qe/quarkus-test-framework/issues/641 and observed in https://github.com/quarkus-qe/quarkus-test-suite/pull/971.

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

File: sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/XAPostgresIT.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class XAPostgresIT extends AbstractSqlDatabaseIT {
 
     static final int POSTGRESQL_PORT = 5432;
 
-    @Container(image = "${postgresql.also.latest.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
+    @Container(image = "${postgresql.latest.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
```
Comment (fedinskiy) [inline sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/XAPostgresIT.java:14]: CI says `XAPostgresIT » NotFound Status 404: {"message":"No such image: postgres:15"}`

**Review by fedinskiy — CHANGES_REQUESTED** [review]

**Comment by rsvoboda** [comment]
@jsmrcka pls rebase and check the failures

**Comment by pjgg** [comment]
I think that this error `"message":"No such image: postgres:15"` is still happening, but currently is hidden by Hibernate 6.2 integration issues

```
ERROR: relation "hibernate_sequence" does not exist
```

**Comment by pjgg** [comment]
@jsmrcka could you rebase this branch with the latest changes?

**Comment by pjgg** [comment]
The error is on test framework on this class `DockerUtils#removeImageById`

Basically is a run condition as far as I see, between these two scenarios `PostgresqlDatabaseIT` , `XAPostgresIT` and the global property `ts.database.container.delete.image.on.stop=true`

As a workaround, I am going to avoid deleting portgresql image on this module.

There is already an open issue on Quarkus test framework talking about this error

https://github.com/quarkus-qe/quarkus-test-framework/issues/641

**Review by pjgg — APPROVED** [review]
LGTM

**Review by fedinskiy — APPROVED** [review]

File: sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/PostgresqlDatabaseIT.java [hunk 2]
```diff
@@ -11,16 +11,18 @@ public class PostgresqlDatabaseIT extends AbstractSqlDatabaseIT {
 
     static final int POSTGRESQL_PORT = 5432;
 
+    // TODO rename "postgresql" database to "database" once this issue is fixed
+    // quarkus-qe/quarkus-test-framework#641
     @Container(image = "${postgresql.latest.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
-    static PostgresqlService database = new PostgresqlService()
+    static PostgresqlService postgresql = new PostgresqlService()
```
[the remark this answers is not in the document]
Comment (pjgg) [reply sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/PostgresqlDatabaseIT.java:17]: That is a good point!

So, I have added the following config

```
ts.postgresql.log.enable=true
ts.postgresql.openshift.use-internal-service-as-url=true
```

[the remark this answers is not in the document]
Comment (jsmrcka) [reply sql-db/sql-app/src/test/java/io/quarkus/ts/sqldb/sqlapp/PostgresqlDatabaseIT.java:17]: Thank you.

**Pull request merged**